internal/provider/scheduler/capacity.go

```diff
@@ -10,6 +10,7 @@ type Capacity struct {
 	MRU uint64
 	SRU uint64
 	HRU uint64
+	CRU uint64
```

Comment:
should get updated in the consume function

Reply:
is it really consumed? i believe cpu is shared. so even if i required 2 cru, does that make the node have 2 cru less???